homeassistant/components/ebox/sensor.py

```diff
@@ -96,9 +156,10 @@ def __init__(self, ebox_data, sensor_type, name):
         """Initialize the sensor."""
         self.client_name = name
         self.type = sensor_type
-        self._name = SENSOR_TYPES[sensor_type][0]
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
-        self._icon = SENSOR_TYPES[sensor_type][2]
+        metadata = SENSOR_TYPES[sensor_type]
+        self._name = metadata.name
```

Comment:
Why not use `attr` variables right away?
```suggestion
        self._attr_name = metadata.name
```

Reply:
done